pages/auth/log-out/index.tsx

```diff
@@ -1,17 +1,35 @@
-import { useEffect } from 'react'
+import  { useEffect } from 'react'
 import { useRouter } from 'next/router'
 import { getLayout } from '@/components/Layout/Layout'
 import { useLogoutMutation } from '@/assets/api/auth/authApi'
-import { PageWrapper } from '@/components/PageWrapper/PageWrapper'
-import { Loading } from '@/components/common/loaders/Loading'
+
+
 
 const LogOut = () => {
+
+
+    const [logout] = useLogoutMutation();
     const router = useRouter()
     useEffect(() => {
-        localStorage.removeItem('accessToken');
-        router.push('/')
-      }, [])
+      const performLogout = async () => {
+        try {
+          await logout({})
+          .unwrap()
+          .then(() => {
+            router.push('/auth/login')
+            localStorage.removeItem('accessToken');
```

Comment:
I`m not sure , that we have to clean accessToken manually, 
I can assume that this is done somehow differently, we need to ask the backend.
It turns out that our request makes no sense if we do it manually